### PR TITLE
Added messages side bar

### PR DESCRIPTION
**Pull request by elayabharath**
image

### CODE REVIEW

File: lib/interactive.dart [hunk 1]
```diff
@@ -7,33 +7,46 @@ import 'package:avf/firebase.dart' as fb;
 Logger logger = Logger("interactive.dart");
 
 const BS_ROW_CSS = "row";
+const BS_CONTAINER_CSS = "container";
```
Comment (lukechurch) [inline lib/interactive.dart]: BS doesn't unpack nicely as an acronym for me - please could you expand

File: lib/interactive.dart [hunk 2]
```diff
@@ -7,33 +7,46 @@ import 'package:avf/firebase.dart' as fb;
...
 // to do: make this configurable based on number of people
 const SQ_IN_ROW = 6;
 const SQ_SIZE = 12;
 
 class Interactive {
   List<model.Filter> _filters;
   List<model.Theme> _themes;
+  List<model.Message> _messages;
   List<model.Person> _people;
+  num _chartWidth;
```
Comment (lukechurch) [inline lib/interactive.dart:56]: In Px presumably?

File: lib/interactive.dart [hunk 3]
```diff
@@ -42,6 +55,22 @@ class Interactive {
     _loadFilters();
   }
 
+  num _gatherChartWidth() {
```
Comment (lukechurch) [inline lib/interactive.dart]: Consider `_computeChartWidth` - gather seems an unusual verb to use here, though give what you need to do I certainly understand how you arrived at it

File: lib/interactive.dart [hunk 4]
```diff
@@ -149,7 +178,7 @@ class Interactive {
 
   html.DivElement _renderChart() {
     var chartWrapper = html.DivElement()..classes = [CHART_WRAPPER_CSS];
-    var chartWidth = _container.offset.width - 2 * CHART_PADDING;
+    var chartWidth = _chartWidth - 4 * CHART_PADDING;
```
Comment (lukechurch) [inline lib/interactive.dart]: Could you use parens please so I don't have to remember the precedence table :)

File: lib/interactive.dart [hunk 5]
```diff
@@ -289,6 +316,17 @@ class Interactive {
     return legendWrapper;
   }
 
+  html.DivElement _renderMessages() {
+    var messagesWrapper = html.DivElement()..classes = [MESSAGES_WRAPPER_CSS];
```
Comment (lukechurch) [inline lib/interactive.dart]: consider `MESSAGES_WRAPPER_CSS_CLASS` - if I understand correctly it's not the full CSS that will get added here but the names of the class?

**Review by lukechurch — APPROVED** [review]

File: lib/interactive.dart (hunk 1 above)
Comment (elayabharath) [reply lib/interactive.dart]: Completely missed it :/ fixed

File: lib/interactive.dart (hunk 2 above)
Comment (elayabharath) [reply lib/interactive.dart:56]: Will fix all the dimensions at one shot

File: lib/interactive.dart (hunk 5 above)
Comment (elayabharath) [reply lib/interactive.dart]: Fixed all occurrences of css to css class

File: lib/interactive.dart (hunk 4 above)
Comment (elayabharath) [reply lib/interactive.dart]: Fixed

File: lib/interactive.dart (hunk 3 above)
Comment (elayabharath) [reply lib/interactive.dart]: Fixed

**Pull request merged**